doc/gb-model/methodology.rst

```diff
@@ -0,0 +1,315 @@
+It is an approach that is stated as suitable for projects with private financing but public benefits.
+
+Delay costs
+-----------
+
+It is possible for options to be delayed from their Earliest In Service Date (EISD).
+This usually comes at a cost, defined by the TO, which is added into the option total cost.
+
+Societal Cost of Carbon
+-----------------------
+
+The societal cost of carbon comes into play if operating carbon emissions reduce as a result of a reinforcement option.
+In such cases, an economic benefit equal to the quantity of reduced emissions (compared to the base case run?) is added to the NPV calculation.
+`Traded carbon values <https://www.gov.uk/government/collections/carbon-valuation--2>`_ are likely used for this calculation.
+
+.. note::
+
+  This does not appear in the NOA 2021-22 methodology, only the tCSNP, so we probably do not need to account for it to reproduce the NOA 2021-22.
+
+Modelling steps
+===============
+
+.. raw:: html
+
+   <iframe src="../_static/existing_methodology_modelling_steps.drawio.html" style="width: 100%; height: 300px; border: none;"></iframe>
+
+   Graphical representation of our understanding of the NOA Refresh 2021-22 modelling steps.
+
+
+The modelling process involves adding candidates one a time in a pre-defined priority order for each specific GB region.
+As :ref:`aforementioned <method-existing-boundary-capabilities>`, reinforcements in other GB regions are fixed based on the most recent NOA report (here: 2020-2021).
+Regions are arbitrarily defined.
+The only criterion is that no option may appear in more than one region.
+
+The method for a region goes as follows:
+
+1. Re-dispatch run is undertaken with all proposed options disabled, to generate a *base case* which defines the reinforcement requirements on each boundary in the region for the first year (2023).
+2. All reinforcement options are ordered by priority to satisfy the requirements (by EISD? by magnitude of requirement met?).
+3. In order of priority, the options are added one-at-a-time and the 20-year re-dispatch model is re-run to get a new set of constraint costs.
+4. If multiple options are equal in priority, they are each modelled independently.
+5. For every option modelled at the same priority level, the NPV (constraint cost relative to base case combined with investment and delay costs) is calculated for introducing it at its EISD and for increasing delay years.
+   The lowest NPV for each option is chosen as its "optimum" year.
+6. If multiple options have been considered, the one with the earliest optimum year is chosen.
+   If it is unclear which should be chosen, the modelling is branched at this point.
+7. The chosen option is added to the base case and aforementioned steps are repeated with a *new* base case and the next option in the priority order.
+8. When the lowest NPV option gives a negative NPV, no more options are considered.
```

Comment:
this is very interesting! I really wonder if that would lead to a global optimum...

Reply:
I doubt it! I do wonder how one might set up a model to assess them all together. Maybe with benders decomposition such that you have a collection of investment decisions (the reinforcement options) as the controller optimisation and rolling horizon dispatch run as the child?